coding-the-legislation/40_legislation_evolutions.md

```diff
@@ -51,76 +53,111 @@ we get the output:
     flat_tax_on_salary:
         2016-01: 500
         2015-12: 400
-        ...
         2015-01: 400
         2014-12: 440
-        ...
         2014-01: 440
 ```
 
 [Read more about how to code parameters](./legislation_parameters.md#parameters-and-time).
 
-## Variable defined only for a specific time intervale
+## Formula evolving over time
+
+Some fiscal or benefit mechanism significantly evolve over time, with bigger changes than a simple parameter adjustement.
 
-As the legislation evolves, some fiscal or benefit mechanism appear and disapear. When creating a variable, you can specify the attribute `start_date` and `stop_date` to defines in which time intervale this variable make sense.
+For instance, let's assume that from the 1st of Jan. 2017, our previous `flat_tax_on_salary` will not be applied on the first `1000` earned by the person.
```

Comment:
For instance, let's assume that starting the 1st of Jan. 2017, the `flat_tax_on_salary` will not be applied on the first `1000` MoneyUnit earned by a person.